mlibrary/get-this
# Updating dependencies.

# Overview
Updating `npm` dependencies for the sake of security.

This needs to be a monthly task.

## Testing
* Remove and install packages (`rm -rf node_modules/ && docker-compose run --rm web npm install`).
* Launch the [site](http://localhost:4567/) and see if everything looks good.
* Try and build the css (`docker-compose run --rm web npm run build`).
